test/test_nn.py

```diff
@@ -4807,6 +4825,9 @@ def assert_weight_allclose_Q(weight, W):
                     opt.step()
                     assert_is_orthogonal(m.weight)
 
+    # FIXME: Rewrite this test using functions not depending on LAPACK
+    #        and remove the `@skipIfNoLapack` (see #70995)
```

Comment:
This one (and the other orthogonal test) are expected to depend on LAPACK, so the FIXME doesn't apply to them.